ansible/roles/postgres/tasks/main.yml

```diff
@@ -49,8 +50,14 @@
     insertafter: "# Put your actual configuration here"
     marker: "# {mark} ANSIBLE MANAGED HBA CONF BLOCK"
     block: |
+      # Manually configured HBA rules
       {% for rule in postgres_hba_rules %}
-      {{ rule.conn_type }}    {{ rule.database }}    {{ rule.user }}    {{ rule.address }}    {{ rule.method }}
+      {{ rule.conn_type }}    {{ rule.database }}    {{ rule.user }}    {{ rule.address }}    {{ rule.method }}   {{ rule['options'] | default('') }}
```

Comment:
Just a nit pick, but we not keep on using dot notation to access attributes ? You have it in the `role_attr_flags` part as well

Reply:
Fixed in force-push, I had a brain error moment re: using default.